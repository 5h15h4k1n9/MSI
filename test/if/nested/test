nestedCondition(3)
nestedCondition(6)
nestedCondition(12)
exit
