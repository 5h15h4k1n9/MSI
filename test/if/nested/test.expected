(string) "small"
(string) "medium"
(string) "large"
Bye-bye!
